### PR TITLE
Added contents of few more members

### PR DESCRIPTION
**Pull request by Subaharish**

### CODE REVIEW

File: functions/views/ACM_Members.ejs [hunk 1]
```diff
@@ -163,38 +163,38 @@
 				</div>
 				<div class="col-12 col-sm-6 col-md-4 col-xl-3 text-center">
 					<div class="card border-0 py-3">
-						<div class="memberImg member">
+						<div class="memberImg adithya">
 							<!-- member image goes here as a background -->
 						</div>
-						<h5 class="card-title">Member Name</h5>
-						<h6 class="card-subtitle mb-2 text-muted">Member Role</h6>
+						<h5 class="card-title">Adithya.S</h5>
```
Comment (aravindvnair99) [inline functions/views/ACM_Members.ejs]:
```suggestion
						<h5 class="card-title">Adithya S</h5>
```

File: functions/views/ACM_Members.ejs [hunk 2]
```diff
@@ -163,38 +163,38 @@
...
 				</div>
 				<div class="col-12 col-sm-6 col-md-4 col-xl-3 text-center">
 					<div class="card border-0 py-3">
-						<div class="memberImg member">
+						<div class="memberImg moulika">
 							<!-- member image goes here as a background -->
 						</div>
-						<h5 class="card-title">Member Name</h5>
-						<h6 class="card-subtitle mb-2 text-muted">Member Role</h6>
+						<h5 class="card-title">Moulika.M.L.S.Chandra</h5>
```
Comment (aravindvnair99) [inline functions/views/ACM_Members.ejs]:
```suggestion
						<h5 class="card-title">Moulika M L S Chandra</h5>
```

File: functions/views/ACM_Members.ejs [hunk 3]
```diff
@@ -163,38 +163,38 @@
...
 				</div>
 				<div class="col-12 col-sm-6 col-md-4 col-xl-3 text-center">
 					<div class="card border-0 py-3">
-						<div class="memberImg member">
+						<div class="memberImg gandheesh">
 							<!-- member image goes here as a background -->
 						</div>
-						<h5 class="card-title">Member Name</h5>
-						<h6 class="card-subtitle mb-2 text-muted">Member Role</h6>
+						<h5 class="card-title">S.A.Gandheesh</h5>
```
Comment (aravindvnair99) [inline functions/views/ACM_Members.ejs]:
```suggestion
						<h5 class="card-title">Gandheesh S A</h5>
```

**Comment by VinayakTekade** [comment]
@Subaharish Fix the names as suggested by @aravindvnair99 and push it. No need to send another pull request, it will reflect here.

**Pull request closed without merging**